### PR TITLE
Add a dockerfile for the repository-api

### LINKED ISSUE #2: Create docker file

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Fixes #2 

Also removes the unused sqllite gem.

## Was the API documentation (openapi.yml) updated?

README update with docker based instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,35 @@ An HTTP API for the SDR.
...
+
+```
+docker-compose run --rm app bundle exec rake db:create
+docker-compose run --rm app bundle exec rake db:migrate
+```
+
+### Start the app
+
+```
+docker-compose run -d app
```
Comment (mjgiarlo) [inline README.md]: Should this be `docker-compose up` or will either work?

Comment (aaron-collier) [reply README.md]: Yeah it should, good catch...

**Comment by aaron-collier** [comment]
@mjgiarlo I figured we'd update circle with a follow up PR.

**Comment by mjgiarlo** [comment]
@aaron-collier :speech_balloon: 
> @mjgiarlo I figured we'd update circle with a follow up PR.

WFM.

File: README.md (hunk 1 above)
Comment (aaron-collier) [reply README.md]: fixed

**Pull request merged**